Gemfile

```diff
@@ -12,6 +12,7 @@ gem 'rails', '~> 5.0.1'
 gem 'pg', '~> 0.18'
 # Use Puma as the app server
 gem 'puma', '~> 3.0'
+gem 'active_model_serializers'
```

Comment:
@akintner Based on the gem instructions, we should include the version number as well. Copy from the gem readme:  
'gem 'active_model_serializers', '~> 0.10.0''